### PR TITLE
docs(readme): added types

### PR DESCRIPTION
**Pull request by harish-sethuraman**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [x] **metadata update**

This is quite a big PR. Adds proper types...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -154,7 +161,13 @@ module.exports = {
 
 ### `import`
 
-Type: `Boolean|Object`
+Type:
+
+```ts
+type import = boolean |
+ { filter:  (url:string, media:string, resourcePath:string) => boolean; }
```
Comment (alexander-akait) [inline README.md:168]: Looks like prettier doesn't work here, because we need space (i.e. ` `) after `:`

Comment (harish-sethuraman) [reply README.md:168]: It should be because `import` is a reserved keyword

Comment (harish-sethuraman) [reply README.md]: I've formatted it now (changed name and formatted and renamed to import)

Comment (alexander-akait) [reply README.md]: Let's use other name and add comment about it

**Review by alexander-akait — APPROVED** [review]

**Review by snitin315 — APPROVED** [review]

**Pull request merged**